bootstrap.js

```diff
@@ -230,8 +230,6 @@ function startup(aData, aReason) {
 function shutdown(aData, aReason) {
   // No need to do extra work here
   Log.debug("shutdown, aReason=", aReason);
-  // Load imports for the BOOTSTRAP_REASONS constants; should have happened before in most cases
-  loadImports();
```

Comment:
Super. Do you want me to merge now or do you want to push a commit that
updates pdf.js?

On Tue, Apr 28, 2015 at 2:01 PM, pianoslum <notifications@github.com> wrote:

> In bootstrap.js
> <https://github.com/protz/GMail-Conversation-View/pull/952#discussion_r29288196>
> :
>
> > @@ -230,8 +230,6 @@ function startup(aData, aReason) {
> >  function shutdown(aData, aReason) {
> >    // No need to do extra work here
> >    Log.debug("shutdown, aReason=", aReason);
> > -  // Load imports for the BOOTSTRAP_REASONS constants; should have happened before in most cases
> > -  loadImports();
>
> I tried install-disable-restart-uninstall and there was no error...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/protz/GMail-Conversation-View/pull/952/files#r29288196>
> .
>